gulpfile.mjs

```diff
@@ -36,15 +36,12 @@ gulp.task('build-sidebar-tailwind-css', () =>
   )
 );
 
-// These CSS source files should not use Tailwind, as they are used in
-// third-party contexts.
-// TODO: Remove `tailwindConfig` after updating `sass` task in `frontend-build`.
-// Setting it is necessary currently to suppress a build warning.
 gulp.task('build-standalone-css', () =>
   buildCSS(
     [
-      // other styles used by annotator (standalone)
+      // styles processed by tailwind, used by annotator
```

Comment:
This allows us to use Tailwind's `sr-only` instead of having to maintain our own screen-reader-only mixin. Otherwise, `highlights.scss` is basic SASS.